package.json

```diff
@@ -27,25 +27,24 @@
   },
   "homepage": "https://github.com/NerdWalletOSS/apollo-invalidation-policies#readme",
   "peerDependencies": {
-    "@apollo/client": "^3.3.0"
+    "@apollo/client": "^3.3.7"
   },
   "dependencies": {
     "@types/graphql": "^14.5.0",
-    "@types/lodash": "^4.14.149",
-    "@types/uuid": "^7.0.2",
-    "graphql": "^15.0.0",
-    "lodash": "^4.17.15",
-    "typescript": "^3.8.3",
+    "@types/lodash": "^4.14.168",
+    "@types/uuid": "^7.0.4",
+    "graphql": "^15.5.0",
+    "lodash": "^4.17.20",
+    "typescript": "^3.9.9",
     "uuid": "^7.0.3"
   },
   "devDependencies": {
-    "@apollo/client": "^3.3.0",
-    "@types/jest": "^25.1.5",
-    "jest": "^25.2.6",
+    "@apollo/client": "^3.3.7",
+    "@types/jest": "^25.2.3",
+    "jest": "^25.5.4",
     "jest-extended": "^0.11.5",
     "prettier": "2.0.5",
-    "react": "^16.13.1",
```

Comment:
Makes sense that we should only include dev dependencies that are really necessary. Is this just an optimization or was having React causing an issue?

Reply:
React was causing an issue since I don't use it.
But I also see it as an optimization since it's not necessary

Reply:
Makes sense, thanks for addressing this.